projects/kubernetes/boot.sh

```diff
@@ -73,7 +73,7 @@ else
     exit 1
 fi
 set -x
-if [ -z "${KUBE_PRESERVE_STATE}" ] ; then
+if [ "${KUBE_CLEAR_STATE}" == "true" ] ; then
```

Comment:
Please use `[ -n "${KUBE_CLEAR_STATE}" ]` so e.g. `KUBE_CLEAR_STATE=y` (which is what I do) works.

Reply:
sorry... should be fine now!